tools/e2e/examples/minikube/kuma-demo/kuma-demo.yaml

```diff
@@ -67,32 +68,22 @@ spec:
       serviceAccountName: demo-app-service-account
       containers:
       - name: demo-app
-        image: "kong:1.1.2-alpine"
+        image: "nginx:latest"
```

Comment:
I know it's unlikely to have ever be a problem with nginx, but maybe we could lock the version here instead of using 'latest'. Wdyt?

Reply:
moved to stable, slightly less moving